docs/windows.md

```diff
@@ -165,8 +135,8 @@ poetry run pre-commit install
 
 Reinstall PyGObject and pycairo using gvsbuild wheels
 ```PowerShell
-poetry run pip install --force-reinstall (Resolve-Path C:\gtk-build\build\x64\release\pygobject\dist\PyGObject*.whl)
-poetry run pip install --force-reinstall (Resolve-Path C:\gtk-build\build\x64\release\pycairo\dist\pycairo*.whl)
+poetry run pip install --force-reinstall (Resolve-Path C:\gtk\wheels\PyGObject*.whl)
+poetry run pip install --force-reinstall (Resolve-Path C:\gtk\wheels\pycairo*.whl)
```

Comment:
Does gvsbuild provide any other wheels, apart from pygobject and pycairo?

Reply:
Nope, just those two.